themes/Atomize-color-theme.json

```diff
@@ -884,6 +884,26 @@
         "foreground": "#C678DD"
       }
     },
+    {
+      "name": "C++ Namespaces",
+      "scope": [
+        "entity.name.scope-resolution.parameter.cpp",
+        "entity.name.scope-resolution.cpp",
+        "entity.name.scope-resolution.function.call.cpp",
+        "entity.name.scope-resolution.template.call.cpp",
+        "entity.name.namespace"
```

Comment:
This will also affect all languages, but might make sense to keep. Do you have examples of what qualifies under this scope? It probably should be moved under `"Class, Types"` above though to keep global modifiers together.

Reply:
Here is the namespaces:
![изображение](https://user-images.githubusercontent.com/22453358/91067598-ccb38f00-e63b-11ea-94af-45b77e7f3364.png)

It's not a type or class, it's just a scope But usually it have the same color as a type / class (see Qt Creator screenshots in #18). Should I move it to the `"Class, Types"`?

Reply:
Yeah I'd move it, feel free to rename the category to something else to make it more general if you want (`"Class, Types, Scopes"` for example)

Reply:
Sounds good, will do!